lib/srv/desktop/rdp/rdpclient/src/rdpdr/consts.rs

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 Gravitational, Inc
```

Comment:
I don't love the idea of a `consts` file that mixes the code from two separate features (smart cards and shared folders).

Can we structure the code in such a way that it's grouped by feature?